src/accelerators/LinStepAccelerator.h

```diff
@@ -41,37 +41,48 @@ int32_t LinStepAccelerator::prepareMovement(int32_t currentPos, int32_t targetPo
     ve_sqr = ve * ve;
     vt_sqr = vt * vt;
 
```

Comment:
Ok, 
`ve != vs` adds a lot of new cases :-).  Looks very good at a first glance but I want to follow that through thoroughly and do some tests.

Reply:
Yes, this one is important for the motion planner to get smooth transitions between the targets. There are more PR's that further enhance the smoothness of target-transitions but I think it's wise to keep the PRs quite small :)

I also believe it needs to be tested for a while before it can be merged.